### ISSUE TITLE
Load Keras model as policy in PPO

### ISSUE THREAD

**Issue by luckeciano** [openai/baselines#432]
Hello everyone,

My objective is to load a keras model as a policy to iterate through using PPO. The idea is that the model is pre-trained with supervised learning and works as a "seed" for reinforcement learning optimization.

I tried to do it inside `ppo1/mlp_policy.py` replacing the policy model `self.pd`  with my model and didn't work (The model can reproduce a Robot Humanoid Kick inside a specific environment that was previously integrated with baselines. In this case, is not reproducing anything). I also tried to create other class that load this model, but again, no success.

The only situation the model works is when I load the keras model in every prediction (in the method `_act`, I load the keras model and predict). But this is very slow and doesn't meet my goals of integrate this model inside the PPO algorithm to optimize the kick motion.

Could anyone help me?

Thanks!

**Comment by AloshkaD** [comment]
@luckeciano have you made any progress on this?

**Comment by feliperafael** [comment]
Hey, I'm going through the same problem. You got any effective solution to create custom policies using tf.keras pretrained models?
